src/tests/functional/inference_engine/serialization/single_layer/nonzero.cpp

```diff
@@ -11,12 +11,19 @@ using namespace LayerTestsDefinitions;
 
 namespace {
 TEST_P(NonZeroLayerTest, Serialize) {
-    Serialize();
+    serialize();
 }
 
-std::vector<InferenceEngine::SizeVector> inputDims = {
-    {7},       {1000},         {3, 5},       {65, 33},       {33, 65},
-    {1, 1000}, {223, 217, 21}, {3, 4, 5, 1}, {3, 4, 1, 5, 1}};
+std::vector<ov::test::InputShape> inputDims = {
+    {{}, {{7}}},
+    {{}, {{1000}}},
+    {{}, {{3, 5}}},
```

Comment:
Let's add some repeat shapes to cover run time parameters cache function, which in future maybe enable in this node.
```suggestion
    {{}, {{3, 5}}, {{2, 4}}, {{3, 5}}},
```
and if possible add some lower bound and upper bound case for dynamic shape cases, take this as example:https://github.com/openvinotoolkit/openvino/blob/master/src/tests/functional/plugin/cpu/single_layer_tests/mvn.cpp#L191

Reply:
added repeated and lower / upper bound shapes to the cpu single_layer tests